lightning/src/ln/features.rs

```diff
@@ -1078,6 +1110,15 @@ mod tests {
 		assert!(!features.requires_basic_mpp());
 		assert!(features.requires_payment_secret());
 		assert!(features.supports_payment_secret());
+
+
+		// Set flags manually
+		let mut features = NodeFeatures::empty();
+		assert!(features.set_optional_feature_bit(55).is_ok());
+		assert!(features.supports_keysend());
+		assert!(features.set_optional_feature_bit(255).is_ok());
+		assert!(features.set_required_feature_bit(256).is_err());
+
```

Comment:
nit: there're just a few extra newlines here.

Reply:
Should be fixed by [992f102](https://github.com/lightningdevkit/rust-lightning/pull/2522/commits/992f1029acbe4c48cb45827dcce8e01d1edc834f)